substrate/runtime/primitives/src/traits.rs

```diff
@@ -371,31 +371,39 @@ pub type HashingFor<B> = <<B as Block>::Header as Header>::Hashing;
 
 /// A "checkable" piece of information, used by the standard Substrate Executive in order to
 /// check the validity of a piece of extrinsic information, usually by verifying the signature.
-pub trait Checkable: Sized + Send + Sync {
-	type Address: Member + MaybeDisplay;
-	type AccountId: Member + MaybeDisplay;
-	type Checked: Member;
-	fn sender(&self) -> &Self::Address;
-	fn check<ThisLookup: FnOnce(Self::Address) -> Result<Self::AccountId, &'static str>>(self, lookup: ThisLookup) -> Result<Self::Checked, &'static str>;
+/// Implement for pieces of information that require some additional context `Ctx` in order to be
+/// checked.
+pub trait Checkable<Ctx>: Sized {
```

Comment:
Avoid unpronouncable names like `Ctx`

Reply:
agreed and fixed https://github.com/paritytech/polkadot/pull/287/commits/70ab21073b63e65347cfdd4e732f9d97d231bd13